service/controller/v2/key/key.go

```diff
@@ -158,19 +158,19 @@ func WorkerSubnetName(customObject providerv1alpha1.AzureConfig) string {
 	return fmt.Sprintf("%s-%s-%s", ClusterID(customObject), virtualNetworkSuffix, workerSubnetSuffix)
 }
 
-func MasterVMSSName(customObject providerv1alpha1.AzureConfig) string {
-	return fmt.Sprintf("%s-master", ClusterID(customObject))
-}
-
-func WorkerVMSSName(customObject providerv1alpha1.AzureConfig) string {
-	return fmt.Sprintf("%s-worker", ClusterID(customObject))
+func MasterInstanceName(customObject providerv1alpha1.AzureConfig, instanceID string) string {
+	return fmt.Sprintf("%s-master-%06s", ClusterID(customObject), instanceID)
 }
 
 // MasterNICName returns name of the master NIC.
 func MasterNICName(customObject providerv1alpha1.AzureConfig) string {
 	return fmt.Sprintf("%s-Master-1-NIC", ClusterID(customObject))
```

Comment:
I know this line is not part of this PR, but is there a reason why master here is written with uppercase M when others are lowercase? If it can also be lowercase, maybe it could be aligned here?

Reply:
No idea.

Reply:
I think we use lowercase for DNS/hostname but names in azure components  are usually uppercase ( as its usual in ms environment)